### PR TITLE
fix: fix QRCode style

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: fix QRCode style |
| 🇨🇳 Chinese | fix: fix QRCode style |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/qrcode/style/index.ts [hunk 1]
```diff
@@ -38,6 +39,9 @@ const genQRCodeStyle: GenerateStyle<QRCodeToken> = (token) => {
         lineHeight: token.lineHeight,
         background: token.QRCodeMaskBackgroundColor,
         textAlign: 'center',
+        [`& > .expired`]: {
```
Comment (afc163) [inline components/qrcode/style/index.ts:42]:
```suggestion
        [`& > ${componentCls}-expired`]: {
```

File: components/qrcode/index.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ const QRCode: React.FC<QRCodeProps> = (props) => {
               {status === 'loading' && <Spin />}
               {status === 'expired' && (
                 <>
-                  <p>{locale.expired}</p>
+                  <p className="expired">{locale.expired}</p>
```
Comment (afc163) [inline components/qrcode/index.tsx:83]: 这里也要改

Comment (li-jia-nan) [reply components/qrcode/index.tsx:83]: 本地已经改好了，我还没推上去

File: components/qrcode/style/index.ts [hunk 3]
```diff
@@ -38,6 +39,9 @@ const genQRCodeStyle: GenerateStyle<QRCodeToken> = (token) => {
         lineHeight: token.lineHeight,
         background: token.QRCodeMaskBackgroundColor,
         textAlign: 'center',
+        [`& > ${componentCls}-expired`]: {
+          color: 'rgba(0, 0, 0, 0.88)',
```
Comment (MadCcc) [inline components/qrcode/style/index.ts:43]: 写个变量到 QRCodeToken

Comment (li-jia-nan) [reply components/qrcode/style/index.ts]: done

**Review by MadCcc — APPROVED** [review]

**Pull request merged**